xline/src/storage/kv_store.rs

```diff
@@ -436,14 +442,15 @@ where
     }
 
     /// Get `KeyValue` start from a revision and convert to `Event`
-    pub(crate) fn get_event_from_revision(
+    pub(crate) fn get_events_from_revision(
         &self,
         key_range: KeyRange,
         revision: i64,
-    ) -> Result<Vec<Event>, ExecuteError> {
+    ) -> Result<(i64, Vec<Event>), ExecuteError> {
```

Comment:
Please comment the meaning of the return value as it's complicated.

Reply:
Fixed